app/components/AdminDropdown/AgencyDropdown.tsx

```diff
@@ -0,0 +1,64 @@
+import { Agency } from "~/gen";
+import { AdminDropdownProps, AdminDropdown } from ".";
+import {
+  AttributeParams,
+  ManagingAgencyAcronym,
+  ProjectTypeCode,
+  AmountMin,
+  AmountMax,
+} from "~/utils/types";
+import { analytics } from "../../utils/analytics";
+import { useSearchParams } from "@remix-run/react";
+
+export interface AgencyDropdownProps
+  extends Pick<AdminDropdownProps, "selectValue"> {
+  agencies: Array<Agency> | null;
+  setAttributeParams: (value: AttributeParams) => void;
+}
+export function AgencyDropdown({
+  selectValue,
+  agencies,
+  setAttributeParams,
+}: AgencyDropdownProps) {
+  const updateManagingAgencyAcronym = (
+    nextManagingAgencyAcronym: ManagingAgencyAcronym,
+  ) => {
+    analytics({
+      category: "Dropdown Menu",
+      action: "Change Managing Agency",
+      name: nextManagingAgencyAcronym,
+    });
+
+    const projectType = searchParams.get("projectType") as ProjectTypeCode;
+    const min = searchParams.get("min") as AmountMin;
```

Comment:
Related to another comment in the `utils/types` file, these are out of scope, no?

Reply:
It is out of scope, they have been removed.